css/style.css

```
@@ -1116,6 +1116,26 @@ h2 a {
   font-size: 16px;
 }
 
+/* h2 -> h3 header links */
+.heading-link::after {
+  content: '#';
+  color: var(--card-fg);
+}
+
+.heading-link {
+  margin-left: 0.3em;
+  font-size: 1rem;
+  text-decoration: none;
+  opacity: 0;
+  transition: opacity 0.2s ease-in-out;
+}
+
+h2:hover .heading-link,
+h3:hover .heading-link,
+.heading-link:is(:hover, :focus, :focus-visible) {

```

```suggestion
:target .heading-link,
.heading-link:is(:hover, :focus, :focus-visible) {
```

We could also show the anchor persistently when a user arrives via deep link into the content, or has clicked on a heading. Can sometimes help you orient after landing on a page.

This is purely stylistic feedback

---

This is great, I have completely forgotten this. Thank you 💖